frontend/src/app/core/setup/globals/onboarding/onboarding_tour_trigger.ts

```diff
@@ -33,14 +33,6 @@ export function detectOnboardingTour():void {
       currentTourPart = '';
       sessionStorage.setItem(onboardingTourStorageKey, 'readyToStart');
 
-      waitForElement('.onboarding-modal .op-modal--close-button', 'body', () => {
```

Comment:
This should probably be replaced by a listener to an outside click.

Reply:
True, replaced it. good catch! 👍🏽